1-org/modules/ml-org-policies/main.tf

```diff
@@ -0,0 +1,262 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+locals {
+
+  ml_boolean_type_organization_policies = toset([
+    #Disable file downloads on new Vertex AI Workbench instances
+    #Control ID: VAI-CO-4.2
+    #NIST 800-53: AC-3 AC-17 AC-20
+    #CRI Profile: PR.AC-3.1 PR.AC-3.2 PR.AC-4.1 PR.AC-4.2 PR.AC-4.3 PR.AC-6.1 PR.PT-3.1 PR.PT-4.1
+    "ainotebooks.disableFileDownloads",
+
+    #Disable root access on new Vertex AI Workbench user-managed notebooks and instances
+    #Control ID: VAI-CO-4.3
+    #NIST 800-53: AC-3 AC-17 AC-20
+    #CRI Profile: PR.AC-3.1 PR.AC-3.2 PR.AC-4.1 PR.AC-4.2 PR.AC-4.3 PR.AC-6.1 PR.PT-3.1 PR.PT-4.1
+    "ainotebooks.disableRootAccess",
+
+    #Disable terminal on new Vertexx AI Workbench instances
+    #Control ID: VAI-CO-4.4
+    #NIST 800-53: AC-3 AC-17 AC-20
+    #CRI Profile: PR.AC-3.1 PR.AC-3.2 PR.AC-4.1 PR.AC-4.2 PR.AC-4.3 PR.AC-6.1 PR.PT-3.1 PR.PT-4.1
+    "ainotebooks.disableTerminal",
+
+    #Restrict public IP access on new Vertex AI Workbench notebooks and instances
+    #Control ID: VAI-CO-4.7
+    #NIST 800-53: AC-3 AC-17 AC-20
+    #CRI Profile: PR.AC-3.1 PR.AC-3.2 PR.AC-4.1 PR.AC-4.2 PR.AC-4.3 PR.AC-6.1 PR.PT-3.1 PR.PT-4.1
+    "ainotebooks.restrictPublicIp",
+
+    #Require automatic scheduled upgrades on new Vertex AI Workbench user-managed notebooks and instances
+    #Control ID: VAI-CO-4.6
+    #NIST 800-53: AC-3 AC-17 AC-20
+    #CRI Profile: PR.AC-3.1 PR.AC-3.2 PR.AC-4.1 PR.AC-4.2 PR.AC-4.3 PR.AC-6.1 PR.PT-3.1 PR.PT-4.1
+    "ainotebooks.requireAutoUpgradeSchedule",
+
+    #Require VPC Connector
+    #Control ID: CF-CO-4.4
+    #NIST 800-53: SC-7 SC-8
+    #CRI Profile: PR.AC-5.1 PR.AC-5.2 PR.DS-2.1 PR.DS-2.2 PR.DS-5.1 PR.PT-4.1 DE.CM-1.1 DE.CM-1.2 DE.CM-1.3 DE.CM-1.4
+    "cloudfunctions.requireVPCConnector"
+  ])
+
+  access_scope                = var.folder_id != "" ? ["under:folders/${var.folder_id}"] : ["under:organizations/${var.org_id}"]
```

Comment:
this is also very good: letting the user choose between applying policies to a folder or org